proceedings-short.bib

```diff
@@ -124,6 +135,7 @@ @string{ iros_2012
 @string{ iros_2013      = "IROS 2013" }
 @string{ iros_2018      = "IROS 2018" }
 @string{ iscis_2007     = "ISCIS 2007" }
+@string{ iser_2006      = "Exp. Robot. IX" }
```

Comment:
Since the symposium is styling themselves as ISER, would it not make more sense to use that as an acronym here?